src/Sylius/Bundle/OrderBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -52,15 +52,4 @@ public function isNumberUsed($number)
             ->getSingleScalarResult() > 0
         ;
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    protected function getQueryBuilder()
```

Comment:
Whole point of this PR to get rid of these because it caused more trouble than benefit. Most of queries are unique anyway and using protected methods for such developer convenience turned out to be more annoying than helpful.